py/datastore_service.py

```diff
@@ -47,9 +48,9 @@ def make_thread_locals(self):
         self.sql_conn = psycopg2.connect("dbname='%s' user='%s' host='%s' password='%s'" % credentials)
         break
       except Exception as e:
-        sys.stderr.write('Failed to connect to database with: %s' % repr(e))
```

Comment:
why? you'd rather not know if it had a problem connecting? this doesnt make sense to me.